core-server/src/main/java/org/orbisgis/orbisserver/control/xml/GetCapabilitiesController.java

```diff
@@ -101,8 +103,8 @@ public Result displayXML(@Parameter("service") String service, @Parameter("versi
                  if(request.equals("GetCapabilities")){
                    try {
                      indexController.GetXMLFromGetCapabilities();
-                   } catch (JAXBException ignored) {
-                     LOGGER.error(I18N.tr("Unable to parse the incoming xml. \nCause : {0}.", ignored.toString()));
+                   } catch (JAXBException e) {
+                     LOGGER.error(I18N.tr("Unable to parse the incoming xml. \nCause : {0}.", e.getMessage()));
```

Comment:
In the OGC standards, exceptions are defined. In the case of GetCapabilities, the exception in the WPS standard is definec [here](http://docs.opengeospatial.org/is/14-065/14-065.html#53), which send you to the document [OGC 06-121r9](https://www.google.fr/url?sa=t&rct=j&q=&esrc=s&source=web&cd=1&ved=0ahUKEwiFqYeDi_TTAhXLXhoKHUZlDYYQFggjMAA&url=https%3A%2F%2Fportal.opengeospatial.org%2Ffiles%2F%3Fartifact_id%3D38867&usg=AFQjCNHmxITnXCBqUrmqS7VsPxS6l7mPHQ&sig2=8gpBLhVKjjMcVyckiNq0HQ&cad=rja), section 7.4.